### ISSUE TITLE
mllm模型训练，一个epoch训练完任务卡住，gpu利用率100%，无法save checkpoint

### ISSUE THREAD

**Issue by llp1992** [modelscope/ms-swift#4680]
2025-06-23T06:11:27.244592630Z {'loss': 0.09736223, 'token_acc': 0.95950156, 'grad_norm': 0.297454, 'learning_rate': 5.061e-05, 'memory(GiB)': 90.96, 'train_speed(iter/s)': 0.067338, 'epoch': 0.99, 'global_step/max_steps': '900/1814', 'percentage': '49.61%', 'elapsed_time': '3h 42m 38s', 'remaining_time': '3h 46m 6s'}

停留在这里，是多节点的训练，gpu利用率100%

![Image](https://github.com/user-attachments/assets/a8d93e04-dfb4-4a01-8c09-358cddfd5c39)

**Comment by Jintao-Huang** [comment]
什么模型 有shell嘛

**Comment by Jintao-Huang** [comment]
尝试ms-swift3.5.2

**Comment by captain7991** [comment]
我也遇到类似问题，求帮忙看看 https://github.com/modelscope/ms-swift/issues/4638

**Comment by JjjjjZzz** [comment]
同样遇到类似问题，基座qwen3或者DeepSeek-R1-Distill都遇到了

**Comment by whalefa1I** [comment]
> 同样遇到类似问题，基座qwen3或者DeepSeek-R1-Distill都遇到了

是不是指定了max epoch

**Comment by Jintao-Huang** [comment]
已经修复了 release/3.5 release/3.6 都可以
